video/out/vo_vaapi_wayland.c

```diff
@@ -37,13 +37,13 @@ struct va_pool_entry {
     VADRMPRIMESurfaceDescriptor desc;
     struct wl_buffer *buffer;
     struct zwp_linux_buffer_params_v1 *params;
-    uint drm_format;
+    uint32_t drm_format;
 };
```

Comment:
A new line here inbetween the struct definitions please.

Reply:
Oops, it seems there was a misunderstanding (maybe I should have commented a line below). I meant a newline between `};` and `struct va_pool {`. You can make the inside of `va_pool_entry` back how it was before if you want to (personally I don't mind it either way).

Reply:
ahhh, ok. Fixed.